cmd/podman/login.go

```diff
@@ -82,6 +82,10 @@ func loginCmd(c *cliconfig.LoginValues) error {
 		server = registryFromFullName(scrubServer(args[0]))
 	}
 
+	if c.Flag("password").Changed {
+		fmt.Println("WARNING! Using --password via the cli is insecure. Please consider using --password-stdin")
```

Comment:
Do we want to send this to STDERR? Maybe use `logrus.Errorf`?

Reply:
I am not sure if this needs to be sent to stderr since this is a warning.

Reply:
Yes it should use stderr, for output.  Not sure about logrus.Error though since that prints out ERROR, I believe, and this really is not an error.

Reply:
Done.